pkgs/development/tools/flamegraph/default.nix

```
@@ -21,7 +21,7 @@ stdenv.mkDerivation {
 
   meta = with stdenv.lib; {
     license = licenses.cddl;
```

```suggestion
    license = with licenses [ asl20 cddl gpl2Plus ];
```
Looking at some of the scripts' headers it seems they come under one of these licenses.